test/Core.Test/Services/AmazonSesMailDeliveryServiceTests.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.Services
+{
+    public class AmazonSesMailDeliveryServiceTests : IDisposable
+    {
+        private readonly AmazonSesMailDeliveryService _sut;
```

Comment:
What does `_sut` stand for here? I notice it is the same on every test for the service instance.

Reply:
I use `sut` as an abbreviation for "System Under Test". A lot of different books and tutorials bring up that term, so you might encounter it out in the wild depending on where you look. I find this naming convention useful, but that's just my experience and preferences. If you have a preferred convention, I'll stick to that.